pkg/operator/ceph/csi/spec.go

```diff
@@ -103,14 +101,13 @@ var (
 	DefaultProvisionerImage = "quay.io/k8scsi/csi-provisioner:v1.6.0"
 	DefaultAttacherImage    = "quay.io/k8scsi/csi-attacher:v2.1.0"
 	DefaultSnapshotterImage = "quay.io/k8scsi/csi-snapshotter:v2.1.1"
-	defaultResizerImage     = "quay.io/k8scsi/csi-resizer:v0.4.0"
+	DefaultResizerImage     = "quay.io/k8scsi/csi-resizer:v0.4.0"
```

Comment:
Why is this exported all of a sudden?

Reply:
This is to keep parity with other variable images. check the above  variables.

Reply:
Hum, it's strange to have it exported but unused (yet?)

Reply:
These variables are exported to set during build time and test changes locally (it was introduced long back)

Reply:
They are overridden with env vars, right? I still have the same question as Seb about why it needs to be capitalized (exported).

Reply:
This was added to set these flags at the go build time. that's why all Image variables are capitalized as these variables need to be called with the package.Variablename

Reply:
got it, a golang reference needed it external to rook